ethexe/observer/src/query.rs

```diff
@@ -358,15 +359,14 @@ impl Query {
 
     pub async fn download_code(
         &self,
-        code_id: CodeId,
-        origin: ActorId,
-        tx_hash: H256,
+        expected_code_id: CodeId,
```

Comment:
Why expected ?

Reply:
because we can't be sure that this is code_id of the code in blob tx, check is performed in underlying call of `fn read_code_from_tx_hash`, but we expected it to be eq to this param